src/components/pages/annual-report-2018/AnnualReportView.js

```diff
@@ -0,0 +1,331 @@
+              </li>
+              <li>
+                <span className="goal-number">228</span>
+                <p>Organizations Vetted</p>
+              </li>
+              <li>
+                <span className="goal-number">6</span>
+                <p>Educational & Social Events</p>
+              </li>
+              <li>
+                <span className="goal-number">50+</span>
+                <p>Casey Research Interviews</p>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div className="annual-report--shelterconnect">
+        <div className="wrapper">
+          <img src={ImgShelterConnectLogo} alt="ShelterConnect" />
+          <p className="annual-report--text">
+            People experiencing homelessness rely on the internet to apply for
+            housing and benefits, schedule appointments with doctors and case
+            managers, search for jobs, and stay in touch with family and
+            friends. It&apos;s a lifeline for getting back on your feet.
+          </p>
+          <p className="annual-report--text">
+            ShelterConnect works with internet service providers to offer free
+            wifi in shelters and transitional housing facilties and offers
+            portable chargers to address battery charging issues.
+          </p>
+          <div className="annual-report--shelterconnect-stats">
+            <h2>In 2018, ShelterConnect expanded wifi installations by 228%</h2>
+            <ul>
+              <li>
+                <span className="goal-number">3000</span>
+                <p>
+                  people now have access to wifi via ShelterConnect, annually
+                </p>
+              </li>
+              <li>
+                <span className="goal-number">600</span>
+                <p>people no have access to wifi via ShelterConnect, nightly</p>
+              </li>
+              <li>
+                <span className="goal-number">200</span>
+                <p>portable phone chargers distributed</p>
+              </li>
+              <li>
+                <span className="goal-number">7</span>
+                <p>
+                  new wifi installations, including 3 new emergency shelters
+                </p>
+              </li>
+              <li>
+                <span className="goal-number">$50</span>
+                <p>cost for wifi in a shelter for an entire month</p>
+              </li>
+              <li>
+                <span className="goal-number">$0.02</span>
+                <p>cost for wifi for someone in a shelter per night</p>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div className="annual-report--askdarcel">
+        <div className="wrapper">
+          <img src={ImgAskDarcelLogo} alt="AskDarcel" />
+          <p className="annual-report--text">
+            AskDarcel is a robust and continously updated online directory for
```

Comment:
Here's also a spot where both this PR and master show "AskDarcel" but the actual live site shows "SF Service Guide".